service/worker/service.go

```diff
@@ -150,6 +150,7 @@ func NewConfig(params *resource.Params) *Config {
 			},
 			MaxWorkflowRetentionInDays: dc.GetIntProperty(dynamicconfig.MaxRetentionDays),
 		},
+		KafkaCfg: params.KafkaConfig,
```

Comment:
nit: We might miss unit tests for this. I suggest adding unit tests so that if new configs are added, there will already be test infrastructure that helps with this kind of miss.